ui/cypress/e2e/sampleControls.cy.js

```diff
@@ -15,8 +15,11 @@ describe('3-click centring', () => {
     );
   });
 
-  it('Each click is rotating the sample by 90 degrees', () => {
+  it.skip('Each click is rotating the sample by 90 degrees', () => {
```

Comment:
Skipping this test for now, as explained, but if you have any clue @walesch-yan, please let me know :pray:

Reply:
Unfortunately, I have no direct answer to this behavior. My guess would be that some configuration mismatch on cypress' side leads to networking issues. This definitely needs to be investigated in more detail.